cfpb/sbl-frontend
# [Resolve Errors & Warnings] Download Validation Report

closes #444 

## Changes
- [feat(Download Validation Report): Implemented in Errors and Warnings](https://github.com/cfpb/sbl-frontend/pull/447/commits/3958fcc4900ea0f1779cb573e1805b623d5ddab8)
- [chore: abstracted away the filing links into a component](https://github.com/cfpb/sbl-frontend/pull/447/commits/a42745fb956da9542402a0af26c2378db58c4bd8)
- **feat: disabled cache when downloading a validation report**
- feat: added _submission id_ to the filename

## How to Test
- Upload a csv with errors or warnings or both
- Go to the Errors or Warnings page 
- click on `Download Validation Report`
- Verify the results

## Screenshot
<img width="643" alt="download validation report" src="https://github.com/cfpb/sbl-frontend/assets/13324863/eaad5e02-6fb7-447d-8338-7ab474bce960">
